index.html

```diff
@@ -0,0 +1,227 @@
+                        personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="snapshot-list content2">
+                        <li class="Company">HTML</li>
+                        <li class="description">CSS</li>
+                        <li class="date">Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+                
+            </div>
+            <div class="gridcontainer">
+                <div>
+                    <img src="./assets/images/Snapshoot4.png" alt="snapshot">
+                </div>
+                <div class="grid-items">
+                    <h2>Tonic</h2>
+                    <ul class="snapshot-list content1">
+                        <li class="Company">CANOPY</li>
+                        <li class="description">Back End Dev</li>
+                        <li class="date">2015</li>
+                    </ul>
+                    <p class="snapshot-text">
+                        A daily selection of privately 
+                        personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="snapshot-list content2">
+                        <li class="Company">HTML</li>
+                        <li class="description">CSS</li>
+                        <li class="date">Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+                
+            </div>
+
+        </div>
+    </Section>
+    <!-- About Section -->
+    <section class="main_containers">
+        <div class="about-container">
+            <div class="minicontainers about1">
+                <h1 class="about-primary-text">About Me</h1>
+                <p class="about-secondary-text">Hello I’m a software developer! I can help you 
+                    build a product , feature or website. 
+                    Look through some of my work and experience! 
+                    If you like what you see and have a project 
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <div class="social_media">
+                    <h2 class="tertiary-text">Let's connect</h2>
+                    <ul class="socialmedia-icons">
+                        <li><a href="#"><img src="./assets/images/github.svg" alt="github"></a> </li>
+                        <li><a href="#"><img src="./assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                        <li><a href="#"><img src="./assets/images/angel list.svg" alt="angellist"></a> </li>
+                        <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
+                    </ul>
+                  </div>
+                  <button class="btn">Get my resume</button>
+            </div>
```

Comment:
[ ] Please note that as explained before you are required to follow the Figma design as much as possible.
- In the About section your design has a border around it which is not present in the Figma design
- The social media icons are a different color or shade to the Figma design
- The `get my resume` button is different from the one in the Figma design

###
> This is your design 
![Screenshot from 2023-06-03 17-05-23](https://github.com/Bettenoch/My_Portfolio/assets/69245974/d73c891a-2c9c-4ca8-9341-c5522f3727f2)

###
> This is the Figma design
![Screenshot from 2023-06-03 17-04-03](https://github.com/Bettenoch/My_Portfolio/assets/69245974/831f2ddd-44db-4d05-ba49-4c5a3e61ef87)